ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java

```diff
@@ -25,15 +25,19 @@
 @JsonSubTypes({})
 public class Command {
   private final String statement;
+  private final boolean isNewQueryIdGeneration;
```

Comment:
```suggestion
  private final boolean useOffsetAsQueryID;
```
It's more descriptive as to what this actually does. If later we make a change that decides queryID before enqueuing, we wouldn't want a boolean with `isNewNEWQueryIDGeneration` 😂